app/globals.css

```diff
@@ -66,6 +66,6 @@
   }
 }
 
-img {
-  max-height: 48px; 
-}
\ No newline at end of file
+.weeklyPickImage {
```

Comment:
m: we shouldn't be adding custom classes to global.css for an isolated area.  We can use tailwind utility classes for that on the component / tag itself.  

remove this from global.css and see updated comment on the component itself.

Reply:
This is a good point. I'll fix this on the next push.